test/external/test_link_nn.py

```diff
@@ -60,7 +60,7 @@ def test_link_train_fake_product_dataset(tmp_path, share_same_time):
             == train_table_input.src_time[index]
         )
 
-    batch_size = 16
```

Comment:
Any reason you want to increase the batch size here?

Reply:
When I increase `eval_k` to 100 there is some issue with `test_link_nn.py`. The `gnn_link.py` code runs fine, but the tests still fail. I'm part way though figuring out how to fix the test! The batch size adjustment may not be needed in the end since this doesn't seem to fix my issue.

Reply:
Got it, ideally, we keep it.